lambdas/service/app.py

```diff
@@ -1678,9 +1687,12 @@ def validate_wait(wait: Optional[str]) -> Optional[int]:
         else:
             raise ValueError
 
+    def validate_version(version: str) -> None:
```

Comment:
We'd still need a comment that 1) explains the need for the function and 2) mentions the reliance on the ordering of keyword arguments. This should have been obvious from our prior, extensive conversations about this.